python/cudf/cudf/tests/test_parquet.py

```diff
@@ -1617,7 +1617,13 @@ def test_parquet_writer_int96_timestamps(tmpdir, pdf, gdf):
     assert_eq(pdf, gdf)
 
     # Write out the gdf using the GPU accelerated writer with INT96 timestamps
-    gdf.to_parquet(gdf_fname.strpath, index=None, int96_timestamps=True)
+    # TODO: store_schema must be false when working with INT96 timestamps
```

Comment:
Actually `store_schema` is False by default so explicitly setting it False is kinda redundant and can be removed.

Reply:
I don't think we need to as `CUDF_EXPECTS` in C++ will throw an exception so unless specifically handling them in Python for some reason, it would be redundant to test for this specific case

Reply:
The Cython bindings layer must handle and propagate the exception. I recommend adding the test, as we have had issues with this in the past.

Reply:
Added